### PR TITLE
Add option to erase subtitles cache file on next boot

### PR DESCRIPTION
**Pull request by fernandog**
@ratoaq2 as you requested some time ago

```
2017-01-27 13:31:03 DEBUG    MAIN :: [de0be9e] Removing subtitles cache file: /home/osmc/Medusa/cache/subliminal.dbm
2017-01-27 13:31:03 DEBUG    MAIN :: [de0be9e] Removing subtitles cache file: /home/osmc/Medusa/cache/application.dbm
```

### CODE REVIEW

**Comment by fernandog** [comment]
can somebody else review/approve this please?

File: medusa/__main__.py [hunk 1]
```diff
@@ -994,6 +995,18 @@ def initialize(self, console_logging=True):
...
 
+            if app.SUBTITLES_ERASE_CACHE:
+                try:
+                    for cache_file in ['application.dbm', 'subliminal.dbm']:
+                        file_path = os.path.join(app.CACHE_DIR, cache_file)
+                        if os.path.isfile(file_path):
+                            logger.debug(u"Removing subtitles cache file: {cache_file}", cache_file=file_path)
+                            os.remove(file_path)
+                except OSError as e:
+                    logger.debug(u"Unable to remove subtitles cache files. Error: {error}", error=e)
```
Comment (p0psicles) [inline medusa/__main__.py]: What's your reasoning here to use debug and not info?

Comment (ratoaq2) [inline medusa/__main__.py]: I would say it should be a warning

Comment (p0psicles) [inline medusa/__main__.py]: Yeah that was also my first impression. But with a warning, a user should know from the warning which action it should take.

Comment (p0psicles) [inline medusa/__main__.py]: Then the debug before that better be an info.

**Pull request merged**